core/src/main/java/com/linecorp/armeria/common/util/AsyncLoaderBuilder.java

```diff
@@ -82,6 +96,26 @@ public AsyncLoaderBuilder<T> expireIf(Predicate<? super T> expireIf) {
         return this;
     }
 
+    /**
+     * Refreshes the loaded value after the given duration since it was loaded.
+     */
+    public AsyncLoaderBuilder<T> refreshAfterLoad(Duration refreshAfterLoad) {
```

Comment:
Question) It wasn't clear to me how `refreshAfterLoad` differs from `expireAfterLoad`. Do you mind leaving a comment explaining the difference? (and optionally documenting it)

Reply:
- `refreshAfterLoad` triggers the loader to refresh the value before the cache expires to prevent cache misses even though the value is valid. For example, if the expiry time of a token is 30 minutes, we can set `refreshAfterLoad()` to 20 minutes to refresh the token 10 minutes before expiration.
- `expireAfterLoad` indicates the cached value is no longer valid, so it should be re-loaded.

I believe `refreshAfterLoad` would be preferred in most cases.

Reply:
I'm not sure why we introduced another `refreshAfterLoad` here. 

 > refreshAfterLoad triggers the loader to refresh the value before the cache expires to prevent cache misses even though the value is valid. For example, if the expiry time of a token is 30 minutes, we can set refreshAfterLoad() to 20 minutes to refresh the token 10 minutes before expiration.
 
 Can't we just set the expire to 20 minutes in this case?